sympy/polys/polytools.py

```diff
@@ -2669,6 +2676,140 @@ def subresultants(f, g):
 
         return list(map(per, result))
 
+    def subresultant_polys(f, g):
+        """
+        Computes the subresultant polynomials of two polynomials ``f`` and ``g``.
+        The returned list has length min(deg(f), deg(g))+1, and is in order of
+        increasing degree. The first polynomial in the list is the resultant.
+
+        This uses the subresultant PRS. The remainder r_i is the deg(r_{i-1})-th
+        subresultant polynomial. Additionally, if deg(r_i) < deg(r_{i-1}) - 1,
+        then the deg(r_i) subresultant polynomial is r_i * LC(r_i) ^ c_i, where
+        c_i = deg(r_{i-1})-deg(r_i)-1.
```

Comment:
Things like `r_i * LC(r_i) ^ c_i` either need to be rendered as code (double backticks) or LaTeX (single backticks or dollars).